tests/test_tocdirective/test_toc_latex_parts.tex

```diff
@@ -10,83 +10,47 @@
 
 \sphinxAtStartPar
 \sphinxstylestrong{Table of Contents}
-\begin{itemize}
-\item{} 
-\sphinxAtStartPar
-A section
-
-\begin{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{subfolder/index::doc}]{\sphinxcrossref{Subfolder}}}
-
-\begin{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{subfolder/asubpage::doc}]{\sphinxcrossref{Asubpage}}}
-
-\end{itemize}
-\end{itemize}
-\item{} 
-\sphinxAtStartPar
-Another section
-
-\begin{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{content1::doc}]{\sphinxcrossref{Content1}}}
-
-\begin{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{subfolder/asubpage::doc}]{\sphinxcrossref{Asubpage}}}
 
-\end{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{content2::doc}]{\sphinxcrossref{Content2}}}
 
-\begin{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{subfolder/asubpage::doc}]{\sphinxcrossref{Asubpage}}}
-
-\end{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{content3::doc}]{\sphinxcrossref{Content3}}}
-
-\begin{itemize}
-\item{} 
-\sphinxAtStartPar{\hyperref[\detokenize{subfolder/asubpage::doc}]{\sphinxcrossref{Asubpage}}}
-
-\end{itemize}
-\end{itemize}
-\end{itemize}
+\part{A section}
```

Comment:
Here the part was missing, in a toc that contains parts with `jupyterbook-latex==0.2.0`

Reply:
done